README.md

```diff
@@ -132,6 +132,24 @@ yarn build
 
 It builds the app for production to the `build` folder. It correctly bundles React in production mode and optimizes the build for the best performance. The build is minified and the filenames include the hashes.
 
+### 🧪 Test App Locally (E2E with Cypress)
+
+Use the following command to run cypress locally:
+
+```bash
+npm run cypress:open
+```
+
+Or
+
+```bash
+yarn cypress:open
+```
+
+It will open cypress dashboard, through which you need to select `E2E Testing`.
```

Comment:
I would suggest defining a boundary about "who should run these tests" and "when should it be run"

Reply:
Anyone can run this test locally we can not restrict anyone, and there is no particular scenario in which we need to run the test as we are going to integrate GitHub workflow, which will take care of running cypress in every PR